src/api/app/views/webui/kiwi/images/_repository_fields.html.haml

```diff
@@ -1,30 +1,45 @@
-.kiwi-repository-container
-  .nested-fields
-    .triple
-      .field
-        = f.label :repo_type, 'Type:'
-        = f.select :repo_type, Kiwi::Repository::REPO_TYPES
-      .field
-        = f.label :priority, 'Priority:'
-        = f.number_field :priority, maxlength: 4
-      .field
-        = f.label :alias, 'Alias:'
-        = f.text_field :alias
-    .single
-      .field
-        = f.label :source_path, 'Source:'
-        = f.text_field :source_path
-    .double
-      .field
-        = f.label :username, 'User:'
-        = f.text_field :username
-      .field
-        = f.label :password, 'Password:'
-        = f.text_field :password
-    %p
-      = f.check_box :prefer_license
-      = f.label :prefer_license
-      = f.check_box :imageinclude
-      = f.label :imageinclude, 'Image include'
-      = f.check_box :replaceable
-      = f.label :replaceable
+.nested-fields.kiwi_fields
+  .kiwi_list_item
+    %span.kiwi_repository_name= f.object.name
+    %span.kiwi_actions.hidden
+      = link_to sprite_tag("drive_edit", title: 'Edit Repository'), '#', class: "repository_edit"
+      = link_to_remove_association(sprite_tag("req-decline", title: 'Delete Repository'), f)
+
+  .dialog.darkgrey_box{ style: "width: 500px; left: 45%;", class: "#{'hidden' if f.object.source_path.present?}" }
+    .box.box-shadow
+      %h2.box-header= "#{f.object.source_path.present? ? 'Edit' : 'Add'} repository"
+
+      .dialog-content
+        %p
+          = f.hidden_field :order
+          = f.label :repo_type, 'Type:'
+          = f.select :repo_type, Kiwi::Repository::REPO_TYPES
+          = f.label :priority, 'Priority:'
+          = f.number_field :priority, maxlength: 4, style: "width: 50px", in: 0...99
+          = f.label :alias, 'Alias:'
+          = f.text_field :alias,  style: "width: 170px"
+        %p
+          = f.label :source_path, 'Source:'
+          = f.text_field :source_path
+
+        %p
+          = f.label :username, 'User:'
```

Comment:
I guess the username and password attributes are coming from the Kiwi documentation? AFAIK this is not possible with OBS so I would consider dropping it?

cc @hennevogel @adrianschroeter